dashboard/assets/packages/ui/src/components/Stat/Stat.tsx

```diff
@@ -38,16 +45,17 @@ export default function Stat({ panel }) {
   }
 
   const serie = digest.samples.query(query)
-  var value = 0
+  let value: string | undefined
+
   if (serie && Array.isArray(serie.values) && serie.values.length != 0) {
     value = format(serie.unit, Number(serie.values.slice(-1)), true)
   }
 
-  let options = {
+  const options: Options = {
     width: width,
     height: 32,
     title: value,
-    series: plot.series,
+    series: plot.series as Series[],
```

Comment:
Perhaps we could find a way for SeriesPlot to return the correct type? Type 'object | undefined' is not assignable to type 'Series[]'